frontend/src/metabase/admin/settings/components/UploadSettings/UploadSettingsForm.tsx

```diff
@@ -290,6 +290,19 @@ const H2PersistenceWarning = () => (
       <Text>
         {t`Warning: uploads to the Sample Database are for testing only and may disappear. If you want your data to stick around, you should upload to a PostgreSQL or MySQL database.`}
       </Text>
+      <Tooltip
+        label={
+          <>
+            <Text mb="md">{t`By enabling uploads to the Sample Database, you agree that you will not upload or otherwise transmit any individually identifiable information, including without limitation Personal Data (as defined by the General Data Protection Regulation) or Personally Identifiable Information (as defined by the California Consumer Privacy Act and California Privacy Rights Act).`}</Text>
+            <Text>{t`Additionally, you acknowledge and agree that the ability to upload to the Sample Database is provided “as is” and without warranty of any kind, and Metabase disclaims all warranties, express or implied, and all liability in connection with the uploads to the Sample Database or the data stored within it.`}</Text>
+          </>
+        }
+        position="bottom"
+        multiline
+        maw="30rem"
+      >
+        <Text span underline weight={700}>{t`Additional terms apply.`}</Text>
+      </Tooltip>
```

Comment:
according to the product doc, this should only appear for metabase cloud hosted instances. We're not responsible for where people store data on their own infrastructure.

Reply:
Well, this is an embarrassing oversight on my side.
Thank you for catching it.